tests/ttnn/unit_tests/operations/test_reduce_scatter_post_commit.py

```diff
@@ -75,12 +76,12 @@ def run_reduce_scatter_test(
 
     # Run the op
     # for i in range(num_iters):
-    tt_out_tensors = ttl.tensor.reduce_scatter(
+    tt_out_tensors = ttnn.reduce_scatter(
         tt_input_tensors,
```

Comment:
I assume @cfjchu will have the same request here to use the new mesh tensor infrastructure. I view it as somewhat orthogonal to moving the ops to TTNN so I personally would not hold up the PR and would just make sure to do that  after the move is done.

Reply:
Yes, as long as it is not dropped. The primary way that users should be using the op should be via multi-device tensors so I'd expect this change to be done so it's usable.

Reply:
I'm trying this here: https://github.com/tenstorrent/tt-metal/issues/10296